CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ add_custom_target(find_version
                 -DREV_MAJOR=${PROJECT_VERSION_MAJOR}
                 -DREV_MINOR=${PROJECT_VERSION_MINOR}
                 -DREV_BUILD=${PROJECT_VERSION_PATCH}
+                -DWINDOWS=${WIN32}
```

Comment:
Does `-DWIN32=${WIN32}` work instead, and remove the need for the second change?

Reply:
Yes, that works too.